highcharts-angular/src/lib/highcharts-chart.component.ts

```diff
@@ -23,10 +24,18 @@ export class HighchartsChartComponent implements OnDestroy, OnChanges {
 
   constructor(
     private el: ElementRef,
-    private _zone: NgZone // #75
+    private _zone: NgZone, // #75
+    private @Inject(PLATFORM_ID) platformId: Object,
   ) {}
 
+  ssrCompatible() {
+    if(this.platformId && isplatFormServer(this.platformId)) {
+      return;
+    }
+  }
+
   ngOnChanges(changes: SimpleChanges): void {
+    this.ssrCompatible();
```

Comment:
This won't work. You need to make it like this:

```ts
ngOnChanges(changes: SimpleChanges): void {
 if (this.ssrCompatible()) {
      return;
    }
...
```

[See in the demo.
](https://jsfiddle.net/BlackLabel/x35syfan/)